src/messages/request.rs

```diff
@@ -47,6 +47,15 @@ pub enum Request {
     },
 
     // --- MutableData ---
+    /// Fetches whole MutableData from the network.
```

Comment:
The documentation should probably stress that during churn this is likely to fail to accumulate and will therefore often receive no response at all.